package.json

```diff
@@ -12,7 +12,7 @@
     "url": "https://github.com/bwinton/SnoozeTabs/issues"
   },
   "devDependencies": {
-    "@kadira/storybook": "^2.21.0",
+    "@kadira/storybook": "^2.35.3",
```

Comment:
We just ran into a bunch of problems with inexact versions of npm modules in the Firefox Style Guide, what do you think about pinning this to exactly "2.35.3"?